setup.py

```diff
@@ -53,6 +53,7 @@ def get_version():
     install_requires=install_requires,
     build_requires=install_requires,
     extras_require=extras_require,
+    setup_requires=['wheel'],
```

Comment:
What is the purpose of this line? Can you remove this since it is unrelated to the changes you've made?

Reply:
It's removed now.